Source/WebKit/WebProcess/WebPage/wc/GraphicsLayerWC.cpp

```diff
@@ -584,6 +591,7 @@ void GraphicsLayerWC::flushCompositingStateForThisLayerOnly()
         update.background.color = backgroundColor();
         if (drawsContent() && contentsAreVisible()) {
             update.background.hasBackingStore = true;
+            update.background.backingStoreSize = WebCore::expandedIntSize(scaled(size(), deviceScaleFactor()));
```

Comment:
I think `size() * deviceScaleFactor()` is simpler than `scaled(size(), deviceScaleFactor())`  .

Reply:
Okay, I'll change it.